### PR TITLE
Makes the detective's revolver standard .357

### PR DESCRIPTION
**Pull request by optimumtact**
:cl: oranges
tweak - The detectives revolver now packs a punch again
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Um? This...just removes the ability to modify the revolver.

**Comment by optimumtact** [comment]
Yeah my bad, I got distracted before I finished

**Comment by Fox-McCloud** [comment]
So detective can two-shot anyone at round-start?

**Comment by optimumtact** [comment]
Yes that is possible

**Comment by Anonmare** [comment]
Det's gun is pretty powerful already tbh

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 1]
```diff
@@ -273,4 +231,4 @@
 	if(!SS.transfer_soul("FORCE", user)) //Something went wrong
 		qdel(SS)
 		return
-	user.visible_message("<span class='danger'>[user.name]'s soul is captured by \the [src]!</span>", "<span class='userdanger'>You've lost the gamble! Your soul is forfiet!</span>")
\ No newline at end of file
+	user.visible_message("<span class='danger'>[user.name]'s soul is captured by \the [src]!</span>", "<span class='userdanger'>You've lost the gamble! Your soul is forfiet!</span>")
```
Comment (Menshin) [inline code/modules/projectiles/guns/projectile/revolver.dm]: Typo ? (forfiet=> forfeit)

Comment (optimumtact) [inline code/modules/projectiles/guns/projectile/revolver.dm]: Nothing to do with me, that's just a line ending change.

Suppose I can fix it as well

**Comment by optimumtact** [comment]
and now it's more powerful

**Comment by CosmicScientist** [comment]
_Power creeeeep._

**Comment by optimumtact** [comment]
It's not powercreep, it's how it used to be.

It may not have been perfectly balanced but goddamn it at least it was interesting.

**Comment by PKPenguin321** [comment]
I'm not feeling this

**Comment by GunHog** [comment]
Both a nerf and a buff posted within the same five minutes, likely in response to the Traitor Detective PR.

The drama must flow!

**Comment by Zilenan91** [comment]
This is great :+1:

**Comment by TheNightingale** [comment]
Does this mean we can do .38/.38/.357/.38/.38/.357 loads, or does it all have to be the same type of bullet?
I'm not entirely convinced the .38 needed a change, but this would be pretty interesting.

**Comment by optimumtact** [comment]
It only takes .357

**Comment by TheNightingale** [comment]
That's probably not a good thing. We don't want Detectives doing 60 brute per hit without a non-lethal option (and telebatons don't count). If you could load it with either, that'd be okay, but _just_ .357s? Also, that means any traitor can save ~7TC by emagging into the Det's locker at roundstart.

**Comment by CosmicScientist** [comment]
You could ask for a taser to be non-lethal.

**Comment by optimumtact** [comment]
The detective is not a cop

**Comment by Niknakflak** [comment]
:-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1:

**Comment by Zilenan91** [comment]
So does this make the detective spawn with .357s in his locker or does he still have the .38s in there?

**Comment by TheNightingale** [comment]
@optimumtact The Detective is also the only one in Security half the time. As satisfying as it is to do 60 brute per shot, this isn't even a buff, it just makes them better at murder and worse at arresting. How about if we all just leave the Det alone for now? Put down the code and step away?

**Comment by Zilenan91** [comment]
You just gotta not mess with the law fam

**Comment by optimumtact** [comment]
I will fix his locker to have some extra .357, that way he doesn't have to bother cargo right away.

**Comment by Zilenan91** [comment]
I think one extra speedloader is a good number. You shouldn't really ever need more than that.

**Comment by TheNightingale** [comment]
One speedloader is fine; it'll encourage Detectives to visit Cargo more often (I run out of ammo fairly frequently, but only after two or three engagements - so there's time to visit an autolathe in between). I'm still not sold on having the revolver .357-only, though:

* Badcurity Detectives can two-shot people they don't like, and would probably shoot more to instakill
* Goodcurity Detectives have to do lethal damage when less-lethal is needed (e.g. petty crooks)
* Ammunition isn't always available, as it relies on Cargo actually hacking the lathe (not always done)
* If merged along with antag Det, they'll never be trusted with a less-lethal weapon like a taser/egun, even as non-antag... so they'll have to use the 60-brute revolver instead
* Lowpop/low Sec Detectives have no other ranged weapon options, so... do 60 brute to things
* It's not as good at taking out eshield users now, since it doesn't stun
* The AI now actually cares about the Detective, and locks them down all the time
* Antags can raid their office when there's no Det and get a free (13TC equivalent) revolver + ammo

If you really want .357 Dets, just remove the backfire chance for when it's reinforced. You'll have to go to the effort of screwdrivering it and getting your own ammo if you want lethals.

**Comment by Zilenan91** [comment]
Actually yeah that's probably a problem. Antags can emag in from maint and heyo free revolver. I think the detective should spawn with his revolver on him so that doesn't happen.

**Comment by TheNightingale** [comment]
Detectives spawning with a .357-caliber revolver means a HoP can open more Detective slots... for more .357 revolvers. Also, with armour as weak as it is right now (buff armour already), 60 brute is a whole lot of damage, and two-shotting Security is something you should have to put the majority of your TC into being able to do.
(This also means antags can get _literally_ a free revolver, by hacking in, stealing the locker and EMPing it open with the EMP grenade in maint. Or for 1TC, just C4 it open.)

**Comment by KorPhaeron** [comment]
>it's how it used to be.

Probably one of my sillier changes. I think it got reverted after a week.

Random greyshirts adminhelping that the detective blew them away for hacking a door was funny.

**Comment by Zilenan91** [comment]
It's a good change, and if a detective really blasts an assistant for hacking a door that dude deserves a ban tbh.

**Comment by Fox-McCloud** [comment]
>It's not powercreep, it's how it used to be.

I kinda doubt that.

Considering the detective's revolver dates back clear to 4407, it's...been weak since that release

**Comment by Lumipharon** [comment]
Honestly this is a nerf in most cases, since the det isn't usually trying to murder people.
All this does is enable murder more, and encourage people to steal the gun for free revolver.

**Comment by optimumtact** [comment]
@Lumipharon 
that sounds interesting 

@Fox-McCloud it used to be a lethal weapon, not a stun one, or at least that's how I remember it.

**Comment by Lumipharon** [comment]
Before the stun nerfs, the det revolver just did a stun and some shit amount of damage. At some point after, it got given stam damage on top of the stun (which duration I think had been decreased), and also got its damage buffed.

**Comment by Dorsisdwarf** [comment]
:-1

Also made the title more descriptive

**Comment by CosmicScientist** [comment]
Imaginably giving the _first_ detective a choice of either a gun that does what guns do which is be lethal or a taser might be a happy spot with the idea that they smuggled it in (it's not standard edition, it's one of their belongings) and that they shouldn't expect to get spare ammunition (or I'd hope cargo wouldn't hand out .357 ammunition to them and be disappointed) or expect any HoS to allow them to keep a lethal weapon.

I'd rather not see players rush with an emag to steal a weapon and ammunition not left out by a player's actual or pretend incompetence.

**Comment by Supermichael777** [comment]
This would actually discourage stealing the gun as it is a strictly inferior version of a gun you can alredy get (6 shots fam).  The detectives gun is the best survival gun in the game.  If there's no detective then you can guarantee its going to end up in the hands of the hos or a traitor.  Its that good.

**Comment by Robustin** [comment]
I thought this was a troll PR

Can't believe anyone is taking this seriously. Detective chasing a suspect, fires down the hall and kills the captain, a sec officer, and a doctor with his missed shots.

Wew lads. Can't wait for the gang/rev rounds where det has a bag full of .357 and is just sprinting down the hall turbomurdering anyone without an implant.

**Comment by Supermichael777** [comment]
theoretically a revolver could be designed to handle any bullet size of the same diameter.
shotguns can handle multiple types and i'm sure we already have a chamber order simulation so it cant be that far feched.
Fun gun fact: this would also imply that the revolver could chamber 9MM Luger(the small arms round of choice for the discerning gunman(almost certainly what SABERS use))
PROOF: https://www.youtube.com/watch?v=UaNoDR1MMCQ
Cue balance concerns (detectives loading the tase and lase near instant death .38 .357 .357 stun and 135 damage)

**Comment by PKPenguin321** [comment]
Muh realism

**Comment by Xhuis** [comment]
Fuck no. :-1:

**Comment by Robustin** [comment]
Yea hilarious it took 2 days here for someone to add "controversial" to giving the detective a 60 brute armor-piercing murderstorm weapon.

I would expect every round to have 1-2 accidental murders from this weapon. "Shooting at a fleeing suspect, miss a shot, hits some assistant 2 screens away that just got up with 55hp after failing to hack a door, assistant goes down and bleeds out because nobody cares" is going to be the standard outcome of this weapon.

**Comment by Dorsisdwarf** [comment]
Controversial refers to player opinions, not "Is it a retarded idea"

**Comment by SweaterKittens** [comment]
:-1: I can't wait to kill a mostly harmless greytider simply because I had no nonlethal ranged option.

**Comment by Core0verload** [comment]
:-1:

**Comment by Zilenan91** [comment]
I still think it's a good idea

**Comment by optimumtact** [comment]
OOC: ForcefulCJS: everyone who spoke out in favor of 
                   detective getting a .357 non-ironically should be banned 
                   from github

**Comment by TechnoAlchemisto** [comment]
I like this a lot, and I think it's okay, although it will make people with access/emags stealing a free 357 a bit bad.

**Comment by Zilenan91** [comment]
Could always make it spawn on the detective, though there might be problems with the HoP opening tons of detective slots and spamming revolvers on the station. I don't think that'd be too bad though.

**Comment by ShadowDeath6** [comment]
Oh god no, you're giving the detective better weapons than anything security can get outside of hacking an autolathe at rounstart

Maybe give him a taser or something at roundstart so he doesn't have incentive to 2shot someone who pets borgs too much.

**Comment by Zilenan91** [comment]
Buckshot is better than the revolver in every way, also what's with the hyperbole.

**Comment by PKPenguin321** [comment]
>Buckshot is better than the revolver in every way, also what's with the hyperbole.

\>uses hyperbole in the same sentence that he asks about what's with the hyperbole

**Comment by Zilenan91** [comment]
It really is though, you can hit somebody with about 3 or 4 pellets at 4 tiles which I believe is slightly above that of a .357 shot. Anything past that is lower, and any closer is higher.

**Comment by Cheridan** [comment]
meme pr; bad idea

**Pull request closed without merging**